### PR TITLE
license: update FSF address v2

### PR DESCRIPTION
**Pull request by jmtaylor90**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata-ids.org/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:

Describe changes:
- rpmlint warned/errored that FSF address was wrong in files so updated to latest GPLv2 boilerplate which has the updated address.
-
-

[PRScript](https://redmine.openinfosecfoundation.org/projects/suricata/wiki/PRscript) output (if applicable):

### CODE REVIEW

File: python/bin/suricatasc [hunk 1]
```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/python
-# Copyright(C) 2013 Open Information Security Foundation
+# Copyright(C) 2019 Open Information Security Foundation
```
Comment (jasonish) [inline python/bin/suricatasc]: Should be 2013-2019.

File: python/suricata/sc/suricatasc.py [hunk 2]
```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/python
-# Copyright(C) 2012 Open Information Security Foundation
+# Copyright(C) 2019 Open Information Security Foundation
```
Comment (jasonish) [inline python/suricata/sc/suricatasc.py]: Same here.

File: qa/prscript.py [hunk 3]
```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env python
-# Copyright(C) 2013, 2014, 2015 Open Information Security Foundation
+# Copyright(C) 2019 Open Information Security Foundation
```
Comment (jasonish) [inline qa/prscript.py]: 2013-2019.

File: python/bin/suricatasc (hunk 1 above)
Comment (jmtaylor90) [reply python/bin/suricatasc]: updated

File: python/suricata/sc/suricatasc.py (hunk 2 above)
Comment (jmtaylor90) [reply python/suricata/sc/suricatasc.py]: updated

File: qa/prscript.py (hunk 3 above)
Comment (jmtaylor90) [reply qa/prscript.py]: updated

**Comment by jmtaylor90** [comment]
continued in #4567

**Pull request closed without merging**